Examples/PublisherExample/Sources/Screens/Map/MapViewController.swift

```diff
@@ -81,33 +81,51 @@ class MapViewController: UIViewController {
     private func setupNavigationBar() {
         title = "Publishing \(trackingId)"
         navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .compose, target: self, action: #selector(onEditButtonPressed))
-        
+
         navigationItem.hidesBackButton = true
         navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Back", style: .plain, target: self, action: #selector(onBackButtonPressed))
     }
-    
+
     // MARK: - Publisher setup
     private func setupPublisher() {
         let resolution = Resolution(accuracy: .balanced, desiredInterval: 5000, minimumDisplacement: 100)
         currentResolution = resolution
-        
+
+        // Authentication to Ably with a private Ably API key
+        let connectionConfiguration = ConnectionConfiguration(apiKey: Environment.ABLY_API_KEY, clientId: "Asset Tracking Publisher Example")
+
+        // Authentication with Ably with an AuthCallback, which is passed a TokenRequest.
+//        let connectionConfiguration = ConnectionConfiguration(clientId: "Asset Tracking Publisher Example") { tokenParams, resultHandler in
```

Comment:
We need to find a way to ensure this code is compiler visible. Perhaps an enum and switch?

Reply:
ok, as far as I understood ... you want to give users an option to select which type of auth they want to use?

Reply:
This is a class I designed. I am confused by this statement too. Maybe you didn't spot that this file is part of the **example** app? looking at `ConnectionConfiguration`, if a user wants to create a `ConnectionConfiguration`, there are multiple convenience initializers to choose on based on their authentication method.

Reply:
My point is that code in a comment will never pass a compiler in CI. It is, therefore, at risk of falling out of sync with interface changes elsewhere and is a maintainability risk for the codebase. We should avoid this.

Reply:
Oh I see what you mean, and in fact, I initially had an enum and switch 😂. However, it just complicated the code so much, and I removed it from the example app. A user would never have an enum or switch to choose between authentication methods. Then I would suggest just removing these lines from the example app.

Reply:
I've removed the commented code and added documentation to the README instead 96109b031b5d285dfb316c71933af2286f47d922